### PR TITLE
Add dispensaries

### PR DESCRIPTION
**Pull request by roxkon**
work in progress with Dispensaries, just want to show what I'm workin on

### CODE REVIEW

**Review by Shalashtein — APPROVED** [review]
These ones look safe

**Comment by Shalashtein** [comment]
The dispensaries and memberships need some planning however. We are adding too many steps between the participant and their goal. If it's all the same for dispensaries, I would rather not have memberships/ After the participant is approved to join a study, they can just look up the closest dispensary to them on the map, visit, and pick up the material without needing to be a member with that specific dispensary. We can have a dispensary model, just to have somewhere to store the extra information specific to dispensaries that we cannot fit into the Profile model.

**Comment by roxkon** [comment]
i removed membership, also added map view

File: config/routes.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@
     resources :users
     resources :pages
     resources :posts
+    resources :dispensaries
```
Comment (Shalashtein) [inline config/routes.rb:8]: We should delete the dispensary routes until we decide what we want to expose, perhaps we won't use all the resources routes

**Review by Shalashtein — COMMENTED** [review]
By the time the changes have been made I'll have edited the portal controllers so we can merge both PRs

**Review by Shalashtein — APPROVED** [review]
Looks good for now, more additions in the future such as selecting how the Dispensary and the user managing it will be associated, and the views related to them

**Pull request merged**